cli/src/cmd/forge/script.rs

```diff
@@ -742,6 +646,167 @@ impl ScriptArgs {
             predeploy_libraries: run_dependencies,
         })
     }
+
+    fn send_transactions(&self, deployment_sequence: &mut ScriptSequence) -> eyre::Result<()> {
+        // The user wants to actually send the transactions
+        let mut local_wallets = vec![];
+        if let Some(wallets) = self.wallets.private_keys()? {
+            wallets.into_iter().for_each(|wallet| local_wallets.push(wallet));
+        }
+
+        if let Some(wallets) = self.wallets.interactives()? {
+            wallets.into_iter().for_each(|wallet| local_wallets.push(wallet));
+        }
+
+        if let Some(wallets) = self.wallets.mnemonics()? {
+            wallets.into_iter().for_each(|wallet| local_wallets.push(wallet));
+        }
+
+        if let Some(wallets) = self.wallets.keystores()? {
+            wallets.into_iter().for_each(|wallet| local_wallets.push(wallet));
+        }
+
+        // TODO: Add trezor and ledger support (supported in multiwallet, just need to
```

Comment:
on this todo, you can reference this: https://github.com/foundry-rs/foundry/blob/a152664164195238db4212b752114dc2c70d94c0/cli/src/opts/mod.rs#L110-L128